src/SoCreate.Extensions.Logging/ActivityLogger/ActivityLoggerExtensions.cs

```diff
@@ -4,31 +4,33 @@ namespace SoCreate.Extensions.Logging.ActivityLogger
 {
     public static class ActivityLoggerExtensions
     {
-        public static void LogActivity<TActivityEnum>(
+        public static void LogActivity<TKeyType, TActivityEnum>(
             this IActivityLogger activityLogger,
             int key,
-            TActivityEnum keyType,
+            TKeyType keyType,
+            TActivityEnum activityType,
             int? accountId,
             int tenantId,
             string message,
             params object[] messageData)
         {
             if (activityLogger == null) throw new ArgumentNullException(nameof(activityLogger));
 
-            activityLogger.LogActivity(key, keyType, accountId, tenantId, null, message, messageData);
+            activityLogger.LogActivity(key, keyType, activityType, accountId, tenantId, null, message, messageData);
```

Comment:
How does this compile with the tenantId here and it is missing from the Interface?